CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+- Turn some `var`s into `const`s. (#6493)
```

Comment:
I don't think this needs a changelog entry

Reply:
Removed.